ScaleformUI_Csharp/MenuPool.cs

```diff
@@ -274,25 +274,19 @@ public async void ProcessMenus(bool draw)
         /// </summary>
         public void CloseAllMenus()
         {
-            if (currentMenu is not null)
+            for (var i = _menuList.Count - 1; i >= 0; i--)
```

Comment:
you can use i == 0 here.. the for will continue until the condition is met

Reply:
It doesn't work, I've tried